kam2doc-lib/src/main/java/it/facile/kam2doc/cropping/CropView.kt

```diff
@@ -0,0 +1,103 @@
+package it.facile.kam2doc.cropping
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.util.AttributeSet
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.widget.FrameLayout
+import androidx.core.view.doOnNextLayout
+import it.facile.kam2doc.InvalidPointsException
+import it.facile.kam2doc.R
+import it.facile.kam2doc.Result
+import it.facile.kam2doc.utils.ImageResizer
+import it.facile.kam2doc.utils.ScanUtils
+import it.facile.kam2doc.utils.cropBitmapFromPoints
+import it.facile.kam2doc.utils.getBitmapFromLocation
+import kotlinx.android.synthetic.main.layout_crop.view.*
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.cancel
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+class CropView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : FrameLayout(context, attrs, defStyleAttr) {
+
+    lateinit var currentImagePath: String
+
+    private val coroutineScope = CoroutineScope(Job())
+    private val currentImage: Bitmap
+        get() = getBitmapFromLocation(currentImagePath)
+    private val polygonPadding: Int
+        get() = polygonView.resources.getDimension(R.dimen.scanPadding).toInt()
```

Comment:
you can remove polygonView
```suggestion
        get() = resources.getDimension(R.dimen.scanPadding).toInt()
```